languages/de.ini

```diff
@@ -835,7 +841,9 @@ Other Authors = "Weitere Verfasser"
 Other Editions = "Weitere Ausgaben"
 Other Libraries = "Weitere Bibliotheken"
 Other Sources = "Weitere Quellen"
+page = "Seite"
```

Comment:
Rather than just translating the word "page" here, I would use a `page_no = "page %%number%%"` translation to accommodate languages where the number might need to come before the label.

Reply:
Sure. You have much more experience on that than I do!

Reply:
The change is just a tiny bit more complicated than that. You would change:

`<?=$this->transEsc('page')?> <?=$page?>`

to:

`<?=$this->transEsc('page_no', ['%%page%%' => $page])?>`

... so rather than displaying first the label and then the page number, we instead pass the page number as a token to the label and let the translator insert it in the appropriate position.

Reply:
Neat! Will do that!

Reply:
Ok, it is done. However, `%%number%%` didn't work but `%%page%%` did. Hope that's correct

Reply:
Thanks, and sorry for giving you incompatible examples -- the placeholder name in the language string needs to match the placeholder name in the template, and I gave you two different values. Glad you figured it out. :-)